### PR TITLE
PAF-259 Bugfix No Error message

### PR DESCRIPTION
**Pull request by TemitopeAyokuHO**
## What?
[ACRS-259](https://collaboration.homeoffice.gov.uk/jira/browse/PAF-259) - Generic errors - No further details about the error message

## Why?
Some field are missing validation error message

## How?
- added validation type error and error messages in validation.js file for fields

## Testing?
Tested locally

## Screenshots (optional)
Screenshot 2024-07-30 at 14 45 43
Screenshot 2024-07-30 at 14 51 38
Screenshot 2024-07-30 at 14 54 47
Screenshot 2024-07-30 at 14 55 07
Screenshot 2024-07-30 at 15 01 18

## Check list

- [x] I have reviewed my own pull request for linting issues (e.g. adding new lines)
- [ ] I have written tests (if relevant)
- [x] I have created a JIRA number for my branch
- [x] I have created a JIRA number for my commit
- [x] I have followed the chris beams method for my commit https://cbea.ms/git-commit/
here is an [example commit](https://github.com/UKHomeOfficeForms/hof/commit/810959f391187c7c4af6db262bcd143b50093a6e)
- [x] Ensure drone builds are green especially tests
- [x] I will squash the commits before merging

### CODE REVIEW

File: yarn-error.log
Comment (jamiecarterHO) [inline yarn-error.log:1]: You can remove this file

File: apps/paf/translations/src/en/validation.json [hunk 1]
```diff
@@ -175,7 +175,8 @@
   },
   "report-person-location-mobile": {
     "maxlength": "You can't use more than {{maxlength}} characters for your answer",
-    "notUrl": "Enter a valid answer"
+    "notUrl": "Enter a valid answer",
+    "internationalPhoneNumber": "Please enter a valid phone number"
```
Comment (jamiecarterHO) [inline apps/paf/translations/src/en/validation.json:179]: Remove the 'Please' and capitalise 'enter'

**Review by jamiecarterHO — CHANGES_REQUESTED** [review]

File: apps/paf/translations/src/en/validation.json (hunk 1 above)
Comment (TemitopeAyokuHO) [reply apps/paf/translations/src/en/validation.json]: Done

File: yarn-error.log
Comment (TemitopeAyokuHO) [reply yarn-error.log:1]: done

File: apps/paf/translations/src/en/validation.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-    "": "You must tell us what the criminals are planning or doing. This will help us to send the information to the right team immediately."
+    "required": "You must tell us what the criminals are planning or doing. This will help us to send the information to the right team immediately."
   },
   "immigration-crime-group": {
-    "": "You must check at least one of the options related to Immigration Crime"
+    "required": "You must check at least one of the options related to Immigration Crime"
   },
   "smuggling-crime-group": {
-    "": "You must check at least one of the options related to Smuggling"
+    "required ": "You must check at least one of the options related to Smuggling"
```
Comment (jamiecarterHO) [inline apps/paf/translations/src/en/validation.json:9]: Theres a space here in `"required "`

**Pull request merged**